src/components/MarketPlace/MarketPlace.jsx

```diff
@@ -217,7 +218,7 @@ export default function MarketPlace(props) {
                         </div>
                     </div>
                 </>
-                : <p>Loading</p>
+                : <PacmanLoader color="var(--green3)" loading={!props.isLoading || !isUserInfoLoading} size={35} className="loader"/>
```

Comment:
Why is loading={!.isLoading}? The naming here is a bit confusing

Reply:
Ah never mind, I see you fixed this in another commit

Reply:
yup, I had the logic there messed up! It should be `loading={props.isLoading || isUserInfoLoading}` instead, I changed that in commit 4